dataprofiler/data_readers/avro_data.py

```diff
@@ -74,18 +81,20 @@ def is_match(cls, file_path, options=None):
 
         # get current position of stream
         if data_utils.is_stream_buffer(file_path):
+            assert not isinstance(file_path, str)
             starting_location = file_path.tell()
 
         is_valid_avro = fastavro.is_avro(file_path)
 
         # return to original position in stream
         if data_utils.is_stream_buffer(file_path):
+            assert not isinstance(file_path, str)
```

Comment:
is this required by `mypy`?

Reply:
It's needed for mypy but I now changed it into an extra condition in the if statement instead of an assertion.